.github/workflows/build-and-publish.yml

```diff
@@ -122,7 +122,7 @@ jobs:
           # Restart the container to apply the config changes
           docker stop ors-instance
           docker container prune -f 
-          docker run -it -d -p 8080:8080 -v $(pwd)/graphs:/home/ors/ors-core/data/graphs -v $(pwd)/conf:/home/ors/ors-conf -e CORS_ALLOWED_ORIGINS=https://example.org --name ors-instance ${{ needs.prepare_environment.outputs.test_image_name }}
+          docker run -it -d -p 8080:8080 -e ORS_CORS_ALLOWED_ORIGINS=https://example.org --name ors-instance ${{ needs.prepare_environment.outputs.test_image_name }}
```

Comment:
Wouldn't it take longer without the mount?
From my understanding, the mount keeps the graph between the container runs and speeds up the second start.

Reply:
Fixed.